reader_test.go

```diff
@@ -369,14 +369,14 @@ func BenchmarkReaderReadRow(b *testing.B) {
 			}
 
 			r := parquet.NewReader(f)
-			rowbuf := make(parquet.Row, 0, 16)
+			rowbuf := make([]parquet.Row, 0, 16)
```

Comment:
Following up from https://github.com/segmentio/parquet-go/pull/160/files#r861234701

`defaultRowBufferSize` is in the `parquet` package, this is a test so I think it's OK to keep as-is.